lib/id3c/cli/command/geocode.py

```diff
@@ -320,7 +322,20 @@ def geocode_address(address: dict) -> dict:
         LOG.warning(f"Missing street address; can't geocode")
         return None
 
-    STREET_CLIENT.send_lookup(lookup)
+    # Smarty Streets sends a no-op alert for us to stderr, so we capture it
+    # and log it to info. Any other stderr output from Smarty Streets is
+    # surfaced through the logging module as an error.
+    smarty_stderr = io.StringIO()
+    with contextlib.redirect_stderr(smarty_stderr):
+        STREET_CLIENT.send_lookup(lookup)
+
+    for smarty_err in smarty_stderr.getvalue().splitlines():
+        if smarty_err.startswith('There was an error processing the request. Retrying in'):
+            LOG.info(smarty_err)
+        else:
+            LOG.error(smarty_err)
+    smarty_stderr.close()
```

Comment:
Nonblocker, but this rerouting of stderr could be nice to put into a short function which could be reused below as well.

Reply:
Ah yeah for sure I will do that, I was looking at it wanting to put some of the code duplication in a function but getting pause from the two different smarty streets calls. Not sure why I didn't just think to put some of the duplication in a function...